srcOA3/mixins/response/relativePaths.yml.njk

```diff
@@ -0,0 +1,10 @@
+application/json:
+  schema:
+    type: object
+    properties:
+      included:
+        $ref: {{ mixinDirectory }}/standard.yml.njk
```

Comment:
the objective here is an ability to shorthand the path to the mixin directory?

Reply:
Yes, so that you can reference files in the mixin and have the references still work after rendering.

I tried just using a mixin in a mixin, but it didn't work.

Reply:
ah ok nice, so this is to really enable nested mixins! Very cool.. i had never even tried that before.